packages/drivers/odsp-driver/src/odspDocumentServiceFactory.ts

```diff
@@ -35,3 +40,70 @@ export class OdspDocumentServiceFactory
         );
     }
 }
+
+export class LocalOdspDocumentServiceFactory extends OdspDocumentServiceFactoryCore {
+    constructor(
+        private readonly fluidFile: Uint8Array | string,
```

Comment:
I'd call it localSnapshot or something like that - I do not think "file" is appropriate here.